src/rhsmlib/dbus/objects/register.py

```diff
@@ -250,7 +336,23 @@ def Register(self, org, username, password, options, connection_options, locale)
             if not org:
                 raise OrgNotSpecifiedException(username=connection_options['username'])
 
+            # Remove 'enable_content' option, because it will not be proceed in register service
```

Comment:
proceed? or processed? I'm not sure I understand

Reply:
Thank. It's typo. It will not be processed is correct.